### ISSUE TITLE
Showing a simple message dialog in fullscreen apps - possible with simple code?

### ISSUE THREAD

**Issue by rolfschr** [jonathanslenders/python-prompt-toolkit#666]
Hi,

I would like to show a message dialog in a full screen application but I can't find anything about this in the docs. I have seen what I want in ``examples/fullscreen/text-editor.py``, specifically the 'About' dialog (I simply need something to confirm). The example contains a lot of code in order to get the dialog box appear and I was hoping to find something more simple like one or two function calls. Is that possible?
I thought ``message_dialog`` would do, but it doesn't due to the 'event loop is already running' exception (similar to #652).

**Comment by techtonik** [comment]
I also look to fix this one:
```
  File "prompt_toolkit/shortcuts/dialogs.py", line 110, in message_dialog
    return _run_dialog(dialog, style, async_=async_)
  File "prompt_toolkit/shortcuts/dialogs.py", line 193, in _run_dialog
    return application.run()
  File "prompt_toolkit/application/application.py", line 682, in run
    return run()
  File "prompt_toolkit/application/application.py", line 655, in run
    run_until_complete(f, inputhook=inputhook)
  File "prompt_toolkit/eventloop/defaults.py", line 123, in run_until_complete
    return get_event_loop().run_until_complete(future, inputhook=inputhook)
  File "prompt_toolkit/eventloop/posix.py", line 57, in run_until_complete
    raise Exception('Event loop is already running')

Exception Event loop is already running
```

**Comment by jonathanslenders** [comment]
Hi @rolfschr,

Unfortunately, there is not really a shortcut to achieve this.
The reason is that the pop-up dialog should become part of the application layout in order to be displayed. There can be only one application running, so you have to add it to your main application. (The 'dialogs' shortcuts won't do that.) Further, the dialog shortcuts start the event loop, this doesn't work in the middle of a running application, and causes the error you had.

You can have a ``FloatContainer`` as the root of the application, and add new ``Float`` objects to it whenever you need them.

```python
from prompt_toolkit.layout import FloatContainer, Float
from prompt_toolkit.widgets import Dialog, Label, Button

def button_handler():
    pass

dialog = Dialog(
        title=title,
        body=Label(text="YOUR_TEXT", dont_extend_height=True),
        buttons=[
            Button(text="BUTTON_TEXT", handler=button_handler),
], with_background=True)

floatcontainer.floats.append(
    Float(content=dialog),  # optionally pass width and height.
)
```

**Comment by rolfschr** [comment]
Hi,

thanks a lot. I was able to get the dialog as you described. Some hints that helped me:
* The ``floatcontainer`` is actually also the object where the "main" content is placed (I first created a new FloatContainer instead of replacing my exising Window object). I checked ``examples/full-screen/simple-demos/floats.py`` and understood that stuff better than.
* I made  ``button_handler()`` contain ``floatcontainer.floats = []`` to remove the button upon user press.
* I also had to use ``layout.focus(floatcontainer)`` after ``floatcontainer.floats.append(...)``
